doc/api/globals.md

```diff
@@ -560,6 +560,23 @@ changes:
 
 A browser-compatible implementation of {Headers}.
 
+## `localStorage`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental.
+
+A browser-compatible implementation of [`localStorage`][]. Data is stored
+unencrypted in the current working directory, with a storage quota of 10 MB.
+The filename of the persisted storage is the basename of the main entrypoint,
+`process.argv[1]`, followed by a dot (`.`), a hash of the entrypoint's absolute
+path and a `.localstorage` extension. If Node.js is run without a file as the
+entrypoint, such as when running the REPL, then the executable's basename is
+used. Any modification of this data outside of the Web Storage API is not
+supported. Enable this API with the [`--experimental-webstorage`][] CLI flag.
```

Comment:
In a worker thread, does the `global.localStorage` point to the same store as the `global.localStorage` in the main thread? Do they use the same session store? I would assume not. This should be documented/tested. If it is and I missed it, forgive me :-)